portals/knowledge-management/src/App.vue

```diff
@@ -85,7 +86,9 @@
     const portalRounding = ref('');
     const categories = ref({});
     const serviceRequests = ref({});
+    const headerLogo = ref({});
```

Comment:
I believe the default value here should be `''` not `{}` as this is a string not an object.

Reply:
Done